homeassistant/components/datetime/services.yaml

```diff
@@ -5,6 +5,6 @@ set_value:
   fields:
     datetime:
       required: true
-      example: "2022/11/01 22:15"
+      example: "2023-10-07T21:35:22.586205"
```

Comment:
I don't think including microseconds in the example is helpful or useful.

Reply:
Indeed, fixed!